### PR TITLE
Update shade maven plugin add minimizeJar properties

### PR DESCRIPTION
**Pull request by lvassa88**
Update shade maven plugin add minimizeJar properties to reduce the size of the generated jar
Add the minimizedJar because need to upload the jar into blazmeter but it's support upload with max size 50MB.

### CODE REVIEW

File: benchmark/pom.xml [hunk 1]
```diff
@@ -151,4 +152,23 @@
...
+            <id>default</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <properties>
+                <minimize>false</minimize>
+            </properties>
+        </profile>
+        <profile>
+            <id>blazemeter</id>
```
Comment (kami619) [inline benchmark/pom.xml:166]: We do not utilize Blaze-meter tooling in our Keycloak Benchmark, @lvassa88 can you please elaborate how would this be useful for the folks who do not use the Blaze-meter in their frameworks ?

Comment (lvassa88) [reply benchmark/pom.xml:166]: I try to use the keycloak benchmark jar on blameter to run test in different situation/timezone, but the jar that you generate is bigger for BlazeMeter tool, so I use the minimizeJar to solve the issue and upload the jar generated with this option and I think other guys can have the same problem.

**Comment by lvassa88** [comment]
Hi @kami619, I have modified the PR to remove the profile and add only the minimizedJar configuration to reduce the sizing of the jar generated.

**Review by ahus1 — CHANGES_REQUESTED** [review]
I tried this branch locally, and I found that once I packaged the JAR file and used it with the `kcb.sh` shell script we provide, I get an error

```
java.lang.ClassNotFoundException: io.gatling.app.Gatling
```

This indicates that the `minimizeJar` option removes the entry class that we depend on. 

Looking at Blazemeter, maybe they require only the scenarios and not the full Scala/Gatling bundle. If you build it locally, there is a file `original-keycloak-benchmark-0.15-SNAPSHOT.jar` in the target folder that contains only the scenarios. Maybe that is just enough for what you need for Blazemeter.

**Pull request closed without merging**